Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[workspace]
+
+# this is for the new crate structure. The legacy code (current CML) still resides in the `rust` directory.
+members = [
+    "core",
+    "crypto",
+    "cip25",
+    "cip25-wasm",
+    "wasm"
+]
+
+# exclude old crate structure to avoid error in it
+exclude = [
```

Comment:
how the pipeline will be working? will we have one for old and one for new structure? we need to verify both old and new crates are compiling and working correctly